Packs/CortexResponseAndRemediation/Playbooks/playbook-Suspicious_certutil_command_line_README.md

```diff
@@ -0,0 +1,87 @@
+This playbook handles "Suspicious certutil command line" alerts.
+
+Playbook Stages:
+
+Analysis:
+
+During the alert analysis, the playbook will perform the following actions:
+
+- Enrich and checks if the URL reputation is suspicious.
```

Comment:
Im missing the part where we are extracting the URL from the command line

Reply:
Indicator Extraction mode = Inline
applies to Command line analysis task

Reply:
Done. added to the playbook description